api/api.go

```diff
@@ -34,7 +34,10 @@ type (
 		IsDir bool   `json:"is_dir,omitempty"`
 	}
 
-	SetupResponse struct{}
+	SetupResponse struct {
```

Comment:
This is incorrect. Lite-engine does not return instance_id and ip_address as response. That is returned by runner api. We need to revert this PR.